### DISCUSSION TITLE
Is Windows 32bit not supported anymore?

### DISCUSSION THREAD

**Discussion by tech189** [duckdb/duckdb#9506] [General]
The Windows 32bit binary is missing from v0.9.0 and v0.9.1 on the releases page, but a [nightly 32bit build](https://github.com/duckdb/duckdb/actions/runs/6673568013/job/18140731149) is still being run successfully. Just wondering whether to drop 32bit from the [Scoop package](https://github.com/ScoopInstaller/Main/blob/master/bucket/duckdb.json) or not.

**Comment by Giorgi** [comment]
Looks like it's not supported: https://discord.com/channels/909674491309850675/921073327009853451/1156252743376314508

**Reply by tech189** [reply]
Could you post a screenshot or quote please?

**Reply by Giorgi** [reply]
![SmartSelect_20231029_215145_Discord.jpg](https://github.com/duckdb/duckdb/assets/580749/00e1e294-90b2-4d38-bebb-f4f21a7071a5)

**Comment by szarnyasg** [comment]
Windows 32-bit is not supported officially anymore:

https://duckdblabs.com/news/2023/10/02/support-policy.html
> We can only support the following platforms: Windows 10+ on x86_64, Mac OS latest, Linux x86_64 & aarch64, mainstream distros under support by distributor (Ubuntu LTS is a safe bet).
